fxa/__main__.py

```diff
@@ -0,0 +1,205 @@
+    parser.add_argument('--bearer',
+                        help='Generate a Bearer token',
+                        dest='bearer',
+                        action='store_true')
+
+    parser.add_argument('--browserid', '--bid',
+                        help='Generate a BrowserID assertion',
+                        dest='browserid',
+                        action='store_true')
+
+    parser.add_argument('--create', '-c',
+                        help='Create a new user',
+                        dest='create',
+                        action='store_true')
+    parser.add_argument('--auth', '-u',
+                        help='User credentials',
+                        dest='auth',
+                        required=False)
+    parser.add_argument('--out', '-o', '-O',
+                        help='Output file',
+                        dest='output_file',
+                        required=False,
+                        default=None)
+
+    parser.add_argument('--verbose', '-v',
+                        help='Display status',
+                        dest='verbose',
+                        action='store_true')
+
+    # Creation args
+    parser.add_argument('--user-salt',
+                        help=('Salt used to calculate the user credentials. '
+                              '(Random by default)'),
+                        dest='fxa_user_salt',
+                        required=False)
+
+    # FxA server configuration
+    parser.add_argument('--account-server',
+                        help='Firefox Account server URL',
+                        dest='account_server_url',
+                        required=False,
+                        default=FXA_API_URL)
+
+    parser.add_argument('--oauth-server',
+                        help='Firefox Account OAuth server URL',
+                        dest='oauth_server_url',
+                        required=False,
+                        default=FXA_OAUTH_URL)
+
+    parser.add_argument('--client-id',
+                        help='Firefox Account OAuth client id.',
+                        dest='client_id',
+                        required=False,
+                        default=DEFAULT_CLIENT_ID)
+
+    parser.add_argument('--scopes',
+                        help='Firefox Account OAuth scopes.',
+                        dest='scopes',
+                        required=False,
+                        default='profile')
+
+    parser.add_argument('--audience',
+                        help='Firefox BrowserID assertion audience.',
+                        dest='audience',
+                        required=False,
+                        default='https://token.services.mozilla.com/')
+
+    parser.add_argument('--duration',
+                        help='Firefox BrowserID assertion duration.',
+                        dest='duration',
+                        required=False,
+                        default='3600')
+
+    parser.add_argument('--prefix',
```

Comment:
As above, I suggest using a slightly more verbose long-option name to make it clear what the prefix is for.